patches/nan/.patches

```diff
@@ -1 +1,2 @@
 api_simplify_scriptorigin.patch
+nan_string_test_alignment.patch
```

Comment:
Is this patch required, if so what impact might it have on downstream modules, is there anything we can on our end to avoid impact downstream?

Reply:
This works around a DCHECK in v8, modules built against release won't be affected.

Reply:
Can we upstream to nan directly then? I assume the same dcheck would be hit with debug node

Reply:
seems worth trying!

Reply:
I'll put it on my todo list then (since I wrote this patch)!

Reply:
Created a new issue here so we can keep track of it 🙂  https://github.com/electron/electron/issues/29284